README.md

```diff
@@ -338,7 +339,16 @@ underscore must be surrounded by at least one digit.
 flt8 = 9_224_617.445_991_228_313
 ```
 
-64-bit (double) precision expected.
+Float values `-0.0` and `+0.0` are valid and should map according to IEEE 754.
+
+Non-finite float values can also be expressed. They are always lowercase.
+
+```toml
+nf1 = +inf # positive infinity
+nf2 = inf  # positive infinity
+nf3 = -inf # negative infinity
+nf4 = nan  # not a number
```

Comment:
IEEE 754 has 2 types of NaNs -- sNaN and qNaN.

It might be worthwhile to mention that which one it is would be implementation dependant?

Reply:
Yeah, and many of each! I'll add a note as you suggest.